<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>430</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>450</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> font-size: 18px;
                    font-weight: bold;
                    padding: 5px;
                    background-color:white;


                </string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>140</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> font-size: 22px;
                                font-weight: bold;
                                padding: 5px;
                                color: #555555;
                                border:none;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_10">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Enter User PIN</string>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_9">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="login_line_edit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 1px solid gray;</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clear_login_pin_button">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"> background-color: rgb(0,
                                                        170, 255);
                                                        color: white;
                                                        border: none;
                                                        padding: 10px 20px;
                                                        text-align: center;
                                                        text-decoration: none;
                                                        font-size: 14px;
                                                        font-weight:bold;
                                                        margin: 4px 4px;
                                                        border-radius: 8px;</string>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_keypad">
      <property name="spacing">
       <number>15</number>
      </property>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_7">
        <property name="styleSheet">
         <string notr="true"> font-size: 18px;
                                        font-weight: bold;
                                        color: white;
                                        background-color:rgb(0, 170, 255);
                                        padding: 10px;
                                        color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_5">
        <property name="styleSheet">
         <string notr="true"> font-size: 18px;
                                        font-weight: bold;
                                        color: white;
                                        background-color:rgb(0, 170, 255);
                                        padding: 10px;
                                        color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_9">
        <property name="styleSheet">
         <string notr="true"> font-size: 18px;
                                        font-weight: bold;
                                        color: white;
                                        background-color:rgb(0, 170, 255);
                                        padding: 10px;
                                        color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true"> font-size: 18px;
                                        font-weight: bold;
                                        color: white;
                                        background-color:rgb(0, 170, 255);
                                        padding: 10px;
                                        color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_1">
        <property name="styleSheet">
         <string notr="true"> font-size: 18px;
                                        font-weight: bold;
                                        color: white;
                                        background-color:rgb(0, 170, 255);
                                        padding: 10px;
                                        color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="styleSheet">
         <string notr="true"> font-size: 18px;
                                        font-weight: bold;
                                        color: white;
                                        background-color:rgb(0, 170, 255);
                                        padding: 10px;
                                        color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_8">
        <property name="styleSheet">
         <string notr="true"> font-size: 18px;
                                        font-weight: bold;
                                        color: white;
                                        background-color:rgb(0, 170, 255);
                                        padding: 10px;
                                        color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_6">
        <property name="styleSheet">
         <string notr="true"> font-size: 18px;
                                        font-weight: bold;
                                        color: white;
                                        background-color:rgb(0, 170, 255);
                                        padding: 10px;
                                        color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="styleSheet">
         <string notr="true"> font-size: 18px;
                                        font-weight: bold;
                                        color: white;
                                        background-color:rgb(0, 170, 255);
                                        padding: 10px;
                                        color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="enter_password_button">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"> background-color:rgb(60, 60, 60);
                                color: white;
                                border: none;
                                padding: 12px 24px;
                                margin: 8px 0px;
                                text-align: center;
                                text-decoration: none;
                                font-size: 14px;
                                font-weight:bold;
                                border-radius: 8px;</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="login_help_button">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"> background-color:rgb(0, 170, 255);
                                color: white;
                                border: none;
                                padding: 12px 12px;
                                margin: 8px 0px;
                                text-align: center;
                                text-decoration: none;
                                font-size: 14px;
                                font-weight:bold;
                                border-radius: 8px;</string>
         </property>
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
